run_filters.py

```diff
@@ -0,0 +1,168 @@
+        # with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+        #     filter_fill_default(input_ptr, filter_meta, output_ptr)
+        #
+        # print >> sys.stderr, "--------------Updating fields--------------------"
+        # input_path = os.path.join(folder_name, "default.csv")
+        # output_path = os.path.join(folder_name, "Update_field.csv")
+        # with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+        #     filter_update_field(input_ptr, filter_meta, output_ptr)
+        #
+        # print >> sys.stderr, "--------------Removing Unnecessary Records--------------------"
+        # input_path = os.path.join(folder_name, "Update_field.csv")
+        # output_path = os.path.join(folder_name, "CleanedPtid_Update.csv")
+        # with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+        #     filter_remove_ptid(input_ptr, filter_meta, output_ptr)
+        #
+        # print >> sys.stderr, "--------------Removing Records without VisitDate--------------------"
+        # input_path = os.path.join(folder_name, "CleanedPtid_Update.csv")
+        # output_path = os.path.join(folder_name, "final_Update.csv")
+        # with open (output_path,'w') as output_ptr, open (input_path,'r') as input_ptr:
+        #     filter_eliminate_empty_date(input_ptr, filter_meta, output_ptr)
+
+    except Exception as e:
+        print "Error in Opening a file"
+        print e
+
+
+    return
+
+def connect_to_redcap(config_path):
+    #Read in the config file. If the config file is missing or the wrong format, exit the program.
+    print config_path
+    try:
+        with open(config_path, 'r') as config_file:
+            config = yaml.load(config_file.read())
+        print "Config Loaded"
+    except:
+        print("Error: Check config file")
+        exit()
+    return config
+
+
+
+# Getting Data From RedCap
+def get_data_from_redcap(folder_name):
+    # Enter the path for filters_config
+    config = connect_to_redcap("filters_config.yaml")
+    token = config.get('token')
+    redcap_url = config.get('redcap_server')
+    redcap_access_api = API(token, redcap_url, 'master.yaml')
+    res = redcap_access_api.export_records(adhoc_redcap_options={
+                                              'format': 'csv'
+                                          })
+    try:
+        rawdata = str(res.content).encode("utf-8")
+        myreader = csv.reader(rawdata.splitlines())
+        try:
+            with open(os.path.join(folder_name, "redcap_intput.csv"),"w") as file:
+                writer = csv.writer(file, delimiter=',')
+                for row in myreader:
+                    writer.writerow(row)
+        except Exception as e:
+            print("Error in Writing")
+            print e
+
+    except:
+        print("Error in CSV file")
+
+    return
+
+
+if __name__ == '__main__':
+    curentdate = datetime.datetime.now().strftime('%m-%d-%Y')
```

Comment:
please fix typo curent -> current 
Also, since you've used underscore in other 'current' names, like 'current_directory', please call this 'current_date'